src/test/java/org/fundacionjala/coding/jimmy/encryptiontest/CryptoTest.java

```diff
@@ -0,0 +1,180 @@
+package org.fundacionjala.coding.jimmy.encryptiontest;
+
+import org.fundacionjala.coding.jimmy.encryption.CesarCrypto;
+import org.fundacionjala.coding.jimmy.encryption.VigenereCrypto;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Test class for testing the methods.
+ */
+public class CryptoTest {
+
+    private static final int POSITIVE = 1;
+    private static final int NEGATIVE = -POSITIVE;
+    private static final String LIMON = "LIMON";
+    private CesarCrypto cesarCrypto;
+    private VigenereCrypto vigenereCrypto;
+
+    /**
+     * Initializes the class.
+     */
+    @Before
+    public void initialize() {
+        cesarCrypto = new CesarCrypto();
+        vigenereCrypto = new VigenereCrypto();
```

Comment:
This is wrong
because you are not using the two instances in all the tests

Consider splitting in two unit test class